blender/arm/material/make_overlay.py

```diff
@@ -30,7 +30,7 @@ def make(context_id):
     arm_discard = mat.arm_discard
     is_transluc = mat_utils.is_transluc(mat)
     parse_opacity = (blend and is_transluc) or arm_discard
-    make_mesh.make_base(con_overlay, parse_opacity=parse_opacity)
+    make_mesh.make_deferred(con_overlay, parse_opacity=parse_opacity)
```

Comment:
I don't think this will work on forward RP.

Reply:
Ok I'll find a workaround